examples/subscription/package.json

```diff
@@ -20,7 +20,7 @@
     "fastify-cors": "^6.0.2",
     "fastify-gql": "^5.0.0",
     "fastify-static": "^4.5.0",
-    "graphql": "^16.0.1",
+    "graphql": "^15.4.0",
```

Comment:
why this downgrade?

Reply:
this was to fix an issue with npm's peer dependency check when using `apollo-server` and the existing `json-graphql-server` both requiring `graphql`